app/templates/views/index.html

```diff
@@ -1,19 +1,20 @@
 {% extends "document_download_template.html" %}
 
 {% block fullwidth_content %}
-  <h1 class="heading-large">Download your document</h1>
-  <p>{{ service_name or '[SERVICE_NAME]'}} has sent you a document.</p>
+  <h1 class="heading-large">You have a file to download</h1>
+  <p>{{ service_name or '[SERVICE_NAME]'}} sent you a file to download. You may need to save it somewhere you can find it.
```

Comment:
Should be `You'll need to save it somewhere you can find it.`

Reply:
I changed that! It's ... not definite they'll need to save it.

Reply:
On the next screen we tell them they will need to save it.